server/set-port.js

```diff
@@ -15,14 +15,25 @@
 require('dotenv').config();
 
 const fse = require('fs-extra');
+const portfinder = require('portfinder');
 
-if (process.env.ASSET_COMPUTE_DEV_PORT) {
-    // read/process package.json
-    const file = '../client/package.json';
-    const pkg = fse.readJSONSync(file, { throws: false });
-    // at this point you should have access to your ENV vars
-    pkg.proxy = `http://localhost:${process.env.ASSET_COMPUTE_DEV_PORT}`;
+const portRange = {
+    port: process.env.ASSET_COMPUTE_DEV_PORT || 3000, // minimum port
+    stopPort: 3333 // maximum port
```

Comment:
How do we determine those port ranges? Is there a logic behind it? 
If not, I would suggest something like port + 1000 (or 2000). Wdyt?

Reply:
Well the way I saw it to be consistant with react and express defaults - react default was 3000 and express was default 9000 so I just started at those ranges and added enough ports to look at for availability - seemed cleaner than if you saw random ports like client 5683 and server 5429

Reply:
Imeant why stop at 3333 ?

Reply:
It looked better than 3999

Reply:
Yeah, why limit it at all? Just use the full available port range...